client/blocks/comment-detail/comment-detail-actions.jsx

```diff
@@ -38,20 +40,7 @@ export const CommentDetailActions = ( {
 
 	return (
 		<div className="comment-detail__actions">
-			{ hasAction( commentStatus, 'like' ) &&
-				<Button
-					borderless
-					className={ classNames( 'comment-detail__action-like', { 'is-liked': commentIsLiked } ) }
-					onClick={ toggleLike }
-				>
-					<Gridicon icon={ commentIsLiked ? 'star' : 'star-outline' } />
-					<span>{
-						commentIsLiked
-							? translate( 'Liked' )
-							: translate( 'Like' )
-					}</span>
-				</Button>
-			}
+			{ hasAction( commentStatus, 'like' ) && <CommentLikes { ...{ siteId, postId, commentId } } />}
```

Comment:
Missing space before the last `}`. Not sure why eslint isn't catching that though. 🤔 

Also, but this is 100% personal preference, I usually go multiline with conditionally rendered components:
```es6
{ hasAction( commentStatus, 'like' ) &&
	<CommentLikes { ...{ siteId, postId, commentId } } />
}
```
(But it doesn't really matter anyway)